src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTagsVisitCount.java

```diff
@@ -123,7 +124,7 @@ public class InputAbstractJavadocNonTightHtmlTagsVisitCount {
      * <Td> </td>
      * <tH> </th>
      * <body> </body>
-     * <colGROUP> </COLgroup>
+     * <colGROUP> </colGROUP>
```

Comment:
restore as it was before, it is mistake on purpose

Reply:
done